ocean_lib/common/ddo/ddo.py

```diff
@@ -122,7 +123,7 @@ def encrypted_files(self):
 
     def assign_did(self, did):
         if self._did:
-            raise AssertionError('"did" is already set on this DDO instance.')
+            raise AquariusError('"did" is already set on this DDO instance.')
```

Comment:
I don't see this as an Aquarius Error. It does not make any call to Aquarius. AssertionError is ok.

Reply:
Ok, I will keep with `AssertionError`. I just thought that the did is resolved via Aquarius. I misunderstood.